api/ausy_encoder.c

```diff
@@ -0,0 +1,128 @@
+#include "ausy_encoder.h"
+#include "deps/json-parser/json.h"
+#include "oc_log.h"
+#include "deps/tinycbor/src/cborjson.h"
+
+static CborError ausy_encode_value_2_cbor(json_value* value, CborEncoder* currentEncoder);
```

Comment:
This declaration is not necessary, since the function has a definition with declaration on line 8